Dockerfile

```diff
@@ -13,6 +13,6 @@ RUN useradd --create-home --shell /bin/bash spfx && \
 
 USER spfx
 
-RUN npm i -g gulp@4 yo @microsoft/generator-sharepoint@1.15.0
+RUN npm i --location=global gulp@4 yo @microsoft/generator-sharepoint@1.15.2
```

Comment:
Why are we using `--location=global` instead of the `-g` or `--global` which are more common?

Reply:
> Why are we using `--location=global` instead of the `-g` or `--global` which are more common?

I just guessed because of this:

<img width="702" alt="image" src="https://user-images.githubusercontent.com/11289124/185490673-d8b522b5-c35a-47c6-916b-abb5f29797f8.png">

Reply:
Understood! I haven't seen it, so definitely good idea to align with the recommended usage. Let's keep it that way.